sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormReadingOrder.java

```diff
@@ -10,6 +10,12 @@
  */
 public final class FormReadingOrder extends ExpandableStringEnum<FormReadingOrder> {
 
+    /**
+     * Creates a FormReadingOrder object.
+     */
+    public FormReadingOrder() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.